modulefiles/gsi_common.lua

```diff
@@ -6,12 +6,13 @@ local netcdf_ver=os.getenv("netcdf_ver") or "4.7.4"
 
 local bufr_ver=os.getenv("bufr_ver") or "11.7.0"
 local bacio_ver=os.getenv("bacio_ver") or "2.4.1"
-local w3emc_ver=os.getenv("w3emc_ver") or "2.9.1"
+local w3emc_ver=os.getenv("w3emc_ver") or "2.9.2"
+local w3nco_ver=os.getenv("w3nco_ver") or "2.4.1"
```

Comment:
Why is is necessary to add `w3nco` to the list of common modules to build with intel2022?

Reply:
@RussTreadon-NOAA `w3nco` was loaded by `nemsio` versions 2.5.2 and earlier, but is no longer, so it now needs to be loaded explicitly.

Reply:
Thanks for the explanation.

Reply:
@DavidHuber-NOAA 
If you are loading `nemsio`, 2.5.4, it depends on `w3emc`.  You no longer need `w3nco`.

Reply:
@aerorahul I removed the w3nco load.  Thanks!  Note there is a placeholder for the new ncdiag library here which provides a local install on Hera presently.  I will update this when the new ncdiag PR is merged and the new tag is installed everywhere.